includes/helpers.php

```diff
@@ -146,7 +146,7 @@ function get_db_prefix( $blog_id ) {
 	global $wpdb;
 
 	if ( $blog_id > 1 ) {
-		$new_db_prefix = $wpdb->base_prefix . $blog_id . '_';
+		$new_db_prefix = $wpdb->prefix . $blog_id . '_';
```

Comment:
`$wpdb->prefix`  should already include the blog number appended to it, why did you chance this to use `prefix`? $base_prefix  is the original prefix assigned in wp_config.php

Reply:
My guess is the rebase didn't catch this change from master for some reason. Thanks for noticing -- I'll change it to match master in using `$base_prefix` :+1: